### DISCUSSION TITLE
100.000 db fölött kötelező az eÁFA?

### DISCUSSION THREAD

**Discussion by Macskafarka** [nav-gov-hu/eVAT#87] [Kérdések és válaszok - Gépi interfész] [answered]
NAV felületen olvasható és előadáson is elhangzott, hogy az áfa bevalláson belüli 100.000 db bizonylat fölött kötelező az eÁFA M2M használata. Meg tudná nekem valaki mondani, hogy hol található ennek a jogszabályi előírása, és mikor lép hatályba?

**Comment by bmatyi94** [comment]
> NAV felületen olvasható és előadáson is elhangzott, hogy az áfa bevalláson belüli 100.000 db bizonylat fölött kötelező az eÁFA M2M használata. Meg tudná nekem valaki mondani, hogy hol található ennek a jogszabályi előírása, és mikor lép hatályba? 

Nem kötelező, hanem a webes verzió nem használható felette, akkor marad a gép-gép kapcsolat. De nem kötelező az M2M használata, csak aki átlépi az említett határt a webes felület neki nem lesz alkalmas.

**Comment by Macskafarka** [comment, marked as answer]
@bmatyi94 , hol olvastad? Van valahol leírás a webes felületről?

**Reply by bmatyi94** [reply]
A NAV bemutatón is elhangzott: https://www.youtube.com/watch?v=BXirDidycqM&feature=youtu.be

**Comment by renced42** [comment]
Kedves @Macskafarka 

A bevallásra vonatkozóan az ÁFA tv. 184. § 2) bekezdés szól. 
A gépi interfész csak akkor kötelező, ha eÁfa rendszert akar valaki igénybe venni és 100.000 db feletti számlája van, tekintettel arra, hogy a webes felületen eddig támogatjuk a szerkesztési tevékenységet a számlákon.

**Comment by Macskafarka** [comment]
@bmatyi94  Tehát, ameddig van ÁNYK nyomtatványra lehetőség, addig nem számít a bizonylat mennyiége, mert ÁNYK felületen
bármekkora  bizonylatszám esetén beadható a bevallás?

**Reply by bmatyi94** [reply]
@Macskafarka igen, így van. Egyenlőre az ÁNYK továbbra is lehetőség, ha valaki nem venné igénybe az eÁFA rendszert. Ha nagy számlaforgalommal rendelkezik akkor vagy ÁNYK vagy eÁFA M2M, döntés az adózónál. Ha 100 ezer db alatti, akkor meg ÁNYK vagy eÁFA web vagy eÁFA M2M. Az számít bevallásnak majd amelyik rendszerből továbbításra kerül a bevallás a NAV felé. Szóval ha ÁNYK verziót adják be, utána az eÁFA verziót már nem lehet adott bevallási időszakon belül. ÁNYK-ban a bizonylatok mennyisége max a melléklapokon, vagy az M-es lapokon jöhet szóba.

**Reply by renced42** [reply]
Kedves @Macskafarka 
Számít, mert az ÁNYK-ban is van limit...
De ha valaki 100.000 db számlát majd egyesével adókódól, a felületen, akkor annak gratulálni fogok személyesen és meghívom egy sörre.... :)

**Comment by Macskafarka** [comment]
A NAV bemutatón elhangzott a 100.000 db, csak az nem, hogy évente, negyedévente havonta, vagy bevallásonként értendő. Ma már a megvan a válasz: bevallásonként.

**Comment by Macskafarka** [comment]
Az ÁNYK limitről nem hallottam. A kérdésem továbbra is az, hogy ennek van-e jogszabályi alapja? Vagy csak amolyan szóbeszéd alapján megy? Áfa tv., vagy Art. paragrafust várok.

**Comment by Macskafarka** [comment]
Átnéztem a 2365 Áfa bevallás (2465 még nem jelent meg) kitöltési útmutatóját, de nincs benne tételszám korlátra való utalás.

**Comment by Macskafarka** [comment]
Halkan jegyzem meg, hogy nem azonos az ÁNYK és az eÁFA bizonylatmennyiségének a számítása. Almát nem hasonlítunk körtével.  ÁNYK-ban nem kell bizonylatonként szerepeltetni a kimenő számlákat, sem a nyugtákat. Tehát lehet, hogy az eÁFA 100 ezer tétele 3 db bejövő számla egyetlen M-es lapon, a többi kimenő számla és pénztárgépes nyugta. Sokan képesek lennének ezt akár a webes felületen is kezelni, pl. szabályokkal.

**Reply by renced42** [reply]
Kedves @Macskafarka 

Szerintem kevered a számla darabszámot a számlán szereplő tételek darabszámával.
A webes felületen 100.000 darab számláig lehet kezelni a számlákat. A számlákon pedig akárhány tétel lehet.

Az hogy ez miért 100.000 db ahhoz nem kell jogszabály. Ez van beállítva a rendszerben, így működik. Ha lesz társadalmi igény rá, hogy ez magasabb szám legyen akkor azt jelzés alapján meg fogjuk vizsgálni.

**Comment by Macskafarka** [comment]
Én úgy gondolom, hogy csak a jogalkotónak van joga bármiféle korlátot állítani.
